### PR TITLE
Add a bit more introduction to readme

### PR DESCRIPTION
**Pull request by mit-mit**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,16 @@
...
+> Note: ffigen only supports parsing `C` headers, not `C++` headers.
+
+This bindings generator can be used to call C code -- or code in another
+language that compiles to C modules that follow the C calling convention --
+such as Go or Rust. For more details, see:
+https://dart.dev/guides/libraries/c-interop
+
+ffigen also has experimental support for calling ObjC and Swift code;
+for details see:
+https://dart.dev/guides/libraries/c-interop
```
Comment (dcharkes) [inline README.md:16]: https://dart.dev/guides/libraries/objective-c-interop

File: README.md [hunk 2]
```diff
@@ -4,7 +4,16 @@
 
 Binding generator for [FFI](https://dart.dev/guides/libraries/c-interop) bindings.
 
-> Note: ffigen only supports parsing `C` headers.
+> Note: ffigen only supports parsing `C` headers, not `C++` headers.
```
Comment (dcharkes) [inline README.md:7]: I believe we should the C++ note, otherwise developers are going to ask questions again.

**Review by dcharkes — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (mit-mit) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (dcharkes) [reply README.md:7]: Oh, you're adding it. I can't read diffs. 😄

**Comment by dcharkes** [comment]
Thanks @mit-mit !

**Pull request merged**